README.md

```diff
@@ -183,11 +184,45 @@ export default class App extends React.Component {
 You can customise the color of the Intercom widget by passing a `color` prop to
 the `Intercom` component.
 
+## Adding a provider
+
+To add a new provider you need to add the following:
+
+#### 1. Create provider file
+
+Create a new provider file at `src/providers/providerName.js` which exports the
+following:
+
+- `domain`: A string of the domain where the provider script is loaded from
+  that will be used in a `preconnect` link.
+- `load`: Function which when called will load and initialize the provider
+  script. It should accept props and use the `providerKey` as the `app_id` or
+  `api_key`. For consistency, it should call a `loadScript` function.
+- `open`: Function which when called will open the provider chat.
+- `close`: Function which when called will close the provider chat.
+
+Import that new file in `src/providers/index.js` and add it to `Providers`.
+
+The name of this file will be the `providerKey` used in the
+`LiveChatLoaderProvider`.
+
+#### 2. Create component
+
+Create a new component in `src/Components/ProviderName/index.js` which
+replicates the chat widget. Do not worry about loading animations as the widget
+will be shown instantly on page load.
```

Comment:
Should we say something like "your fake chat widget should be positioned using a z-index so that it sits immediately above the chat widget that is being replaced?"